odiglet/pkg/kube/utils/utils.go

```diff
@@ -50,3 +51,11 @@ func GetResourceAttributes(podWorkload *workload.PodWorkload, podName string) []
 
 	return attrs
 }
+
+func GetPodExternalURL(ip string, ports []corev1.ContainerPort) string {
+	if ports != nil && len(ports) > 0 {
```

Comment:
I think `ports != nil` is redundant since `len(ports) > 0` already covers this case